packages/toolkit/src/view/catalog/components/tabs/CatalogMainTab.tsx

```diff
@@ -92,11 +88,20 @@ export const CatalogTab = ({
     if (!accessToken) return;
 
     try {
+      const formattedTags = Array.isArray(data.tags)
+        ? data.tags
+        : typeof data.tags === "string"
+          ? data.tags
+              .split(",")
+              .map((tag) => tag.trim())
+              .filter((tag) => tag !== "")
+          : [];
```

Comment:
@thewbuk You keep using the same function, please isolated them and make it resuable